handbook/engineering/README.md

```diff
@@ -241,6 +241,35 @@ The on-call developer is responsible for:
 3. Comment on the GitHub issue and at-mention the PM and anyone listed in the requester field.
 4. If `customer-` labels are applied to the user story, at-mention the [VP of Customer Success](https://fleetdm.com/handbook/customer-success#team).
 
+### Make changes to vulnerabilitie-repo and NVD-repo
+
+NVD-repo and Vulnerability-repo have two code locations that affect them. 
+1. Their own code (within the repository itself).
+2. Code sections in Fleet main repo that is being copied and run.
+
+Merging PRs into NVD-repo and/or Vulnerability-repo or into the relevant code sections in fleet repo, has an immediate effect of releasing and deploying to all customers without QA. 
```

Comment:
Use real repos names.

Reply:
What are the relevant code sections in the Fleet repo? What triggers the releasing and deploying to customers?